app.js

```diff
@@ -1,134 +1,117 @@
   };
-
   this.totalSales = function() {
     for (var i = 0; i < storeHours.length; i++) {
       var randomHourlySale = this.randomHourlySale();
-      salesArray.push(randomHourlySale);
+      this.salesArray.push(randomHourlySale);
       this.total += randomHourlySale;
     }
   };
-
   this.totalSales();
-  this.renderHTML = function() {
-
-    var newUl = document.createElement('ul');
-    var newHeading = document.createElement('h2');
-    newHeading.innerText = name;
-    body.appendChild(newHeading);
-
-    for (var i = 0; i < salesArray.length; i++) {
-      var newLi = document.createElement('li');
-      newLi.innerText = storeHours[i] + ': ' + salesArray[i] + ' cookies';
-      newUl.appendChild(newLi);
-    }
-
-    var TotalLi = document.createElement('li');
-    TotalLi.innerText = 'Total' + ': ' + this.total + ' cookies';
-    newUl.appendChild(TotalLi);
-    body.appendChild(newUl);
-  };
+  storeArray.push(this);
 };
-
-//Table
-//table function
+console.log(storeArray);
 var tabFunction = function() {
   var table = document.createElement('table');
   body.appendChild(table);
-
-// table head
   var tHead = document.createElement('thead');
   table.appendChild(tHead);
   var headRow = document.createElement('tr');
   tHead.appendChild(headRow);
-
   var spacer = document.createElement('th');
   headRow.appendChild(spacer);
-
   for (var i = 0; i < storeHours.length; i++) {
     var tData = document.createElement('th');
     tData.innerText = storeHours[i];
     headRow.appendChild(tData);
   };
-
-  //table body
   var tBody = document.createElement('tbody');
   table.appendChild(tBody);
-
+  var tFoot = document.createElement('tfoot');
+  tFoot.id = 'bob';
+  table.appendChild(tFoot);
   var tabRow = document.createElement('tr');
   tBody.appendChild(tabRow);
-
   newTableRow(firstPike);
   newTableRow(seaTac);
   newTableRow(seaCenter);
   newTableRow(capHill);
   newTableRow(alki);
-
-//footer
-  var tFoot = document.createElement('tf');
-  body.appendChild(tFoot);
 };
 
 function newTableRow(store) {
   var tBody = document.getElementsByTagName('tbody')[0];
   var tabRow = document.createElement('tr');
   tBody.appendChild(tabRow);
-
   var rowName = document.createElement('th');
   tabRow.appendChild(rowName);
   rowName.innerText = store.name;
-
   for (var i = 0; i < storeHours.length; i++) {
     var tData = document.createElement('td');
     tData.innerText = store.salesArray[i];
     tabRow.appendChild(tData);
   };
 }
-//End Table
-
 tabFunction();
-
-//
 var form = document.getElementById('the-form');
+
 function createNewStore(event) {
   event.preventDefault();
   var name = event.target.elements.storeName;
   var minCust = event.target.elements.minCust;
   var maxCust = event.target.elements.maxCust;
   var avgCookieSale = event.target.elements.aveCookies;
-
   if (maxCust < minCust) {
     alert('The max number of customers should not be larger than the min number of customers');
```

Comment:
This is good, but `alert()` boxes are always an annoying disruption to the user experience. Perhaps you might consider making a message box in the HTML instead, whose only purpose is to display error messages when there's an error in user input?